default.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
     <remote name="github" fetch="https://github.com"/>
+    <remote name="coreboot" fetch="https://review.coreboot.org/"/>
```

Comment:
@miczyg1 have you tested this one? Is the url correct?

Reply:
@mek-x Yes I have tested it many times.